src/app/search/search.component.ts

```diff
@@ -191,23 +191,22 @@ export class SearchComponent implements OnInit {
     this.documentService.get(this.terms, this.projects, this.proponents, this.page, this.limit).subscribe(
       data => {
         this.loading = false;
-
-        // Push in 1st call
+        // mem-data
         if (data[0].results) {
+
           data[0].results.forEach(i => {
             this.results.push(i);
           });
+          this.count = (data[0].count || 0);
```

Comment:
I am thinking it may be more straightforward to read if you create a local variable initialized to 0 for each `memCount` and `epicCount` and just assign them the length of each section of documents, if they return anything.

Then, at the very end (line 210-ish) you could kjust do `this.count = memCount + epicCount` without having so many inline conditional assignments.